azurerm/internal/services/servicebus/validate/servicebus_namespace.go

```diff
@@ -0,0 +1,26 @@
+package validate
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+// validation
+func ServiceBusNamespaceName(v interface{}, k string) (warnings []string, errors []error) {
+	value := v.(string)
+
+	if regexp.MustCompile("^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[8|9|aA|bB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$").MatchString(value) {
```

Comment:
Could we use the `validation.IsUUID()` function here rather then roll our own regex?

Reply:
Yes, I was looking at the UUID regex helper first before `validation.IsUUID()` going by your initial comment. Will use this instead